public/scripts/play.js

```diff
@@ -29,122 +28,89 @@ const animations = {
+      gamePlayer = new Player(
+        gamePlayer.x,
+        gamePlayer.y,
+        gamePlayer.width,
+        gamePlayer.height,
+        new Img(
+          gamePlayer.img.src,
+          gamePlayer.img.startRow,
+          gamePlayer.img.startColumn,
+          gamePlayer.img.rows,
+          gamePlayer.img.columns,
+          gamePlayer.img.speed,
+          '',
+          gamePlayer.img.currentRow,
+          gamePlayer.img.currentColumn,
+        ),
+        gamePlayer.username,
+      );
+      // get data about other players from server
+      gamePlayer.draw(ctx, ((canvas.width - compareToPlayer.width) / 2) - compareToPlayer.x + gamePlayer.x, ((canvas.height - compareToPlayer.height) / 2) - compareToPlayer.y + gamePlayer.y);
+    }
+  });
 }
 
-// event listener for start of the movement
-window.addEventListener('keydown', movePlayer);
-
-// event listener for end of the movement
-window.addEventListener('keyup', stopPlayer);
-
-//Marianna
-//key listeners to move the player
-function movePlayer(e) {
-    //!!!!THINK ABOUT STORING BLOCK TYPES IN SOME GLOBAL VARIABLES!!!!
-    //3rd parameter in isBlockCollision
-    if (!playing || player.isDone) return;
-    switch(e.key) {
-        case "A":
-        case "a":
-        case "ArrowLeft":
-            changeAnimation("left");
-            //check for wall collision
-            if (!player.isBlockCollision(map, "left", 1, -player.speed, 0)) {
-                player.x -= player.speed;
-                //check for goal collision
-                if (player.isBlockCollision(map, "left", 2)) {
-                    playerIsDone();
-                }
-            }
-            break;
-        case "D":
-        case "d":
-        case "ArrowRight":
-            changeAnimation("right");
-            if (!player.isBlockCollision(map, "right", 1, player.speed, 0)) {
-                player.x += player.speed;
-                if (player.isBlockCollision(map, "right", 2)) {
-                    playerIsDone()
-                }
-            }
-            break;
-        case "W":
-        case "w":
-        case "ArrowUp":
-            changeAnimation("up");
-            if (!player.isBlockCollision(map, "up", 1, 0, -player.speed)) {
-                player.y -= player.speed;
-                if (player.isBlockCollision(map, "up", 2)) {
-                    playerIsDone()
-                }
-            }
-            break;
-        case "S":
-        case "s":
-        case "ArrowDown":
-            changeAnimation("down");
-            if (!player.isBlockCollision(map, "down", 1, 0, player.speed)) {
-                player.y += player.speed;
-                if (player.isBlockCollision(map, "down", 2)) {
-                    playerIsDone()
-                }
-            }
-            break;
-        default:
-            //no need to update server if player didn't move
-            return;
-    }
-    //add update of server
-    updateServer();
+// Marianna
```

Comment:
This might take a bit to merge cause of eslint just to compare then whether everything is okay just the order different.

Reply:
yeah I ran eslint, no changes done by me here